File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        res = 0
        for i in arr:
            if i%2!=0:
                res += 1
                if res==3:
                    return True
            else:
                res = 0
        return False
        

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int res = 0;
        for(int i:arr){
            if(i%2!=0){
                res += 1;
                if(res==3){
                    return true;
                }
            }else{
                res = 0;
            }
        }
        return false;
    }
}